views/index.ecr

```diff
@@ -6,7 +6,7 @@
     <link rel="stylesheet" href="/css/application.css">
   </head>
 
-  <body>
+  <body onload="document.forms['search'].elements['search'].select();">
```

Comment:
This is only one text field on the page. It would be more convenient just to type and press enter rather then use a mouse to take a focus before. What do you think ? We can move it to a separate JS file too.